pybind11_stubgen/__init__.py

```diff
@@ -443,9 +443,10 @@ def strip_current_module_name(obj, module_name):
             for b in self.base_classes
         ]
         result = [
-            "class {class_name}({base_classes_list}):".format(
+            "class {class_name}({base_classes_list}):{doc_string}".format(
                 class_name=self.klass.__name__,
-                base_classes_list=", ".join(base_classes_list)
+                base_classes_list=", ".join(base_classes_list),
+                doc_string='\n      """{}"""'.format(self.doc_string) if self.doc_string else "",
```

Comment:
use `self.indent()` or `self.INDENT` to ensure indentation match

Reply:
True, thanks. It's fixed now